src/standalone/Soil/boundary_conditions.jl

```diff
@@ -463,13 +508,28 @@ function ClimaLand.boundary_flux(
     FT = eltype(Δz)
     # Approximate K_bc ≈ K_c, ψ_bc ≈ ψ_c (center closest to the boundary)
     # We need to project the center values onto the face space.
-    K_c = Fields.Field(Fields.field_values(Fields.level(p.soil.K, 1)), axes(Δz))
-    ψ_c = Fields.Field(Fields.field_values(Fields.level(p.soil.ψ, 1)), axes(Δz))
+    surface_space = axes(Δz)
+    K_c = get_bottom_surface_field(p.soil.K, surface_space)
+    ψ_c = get_bottom_surface_field(p.soil.ψ, surface_space)
 
     # Calculate pressure head using boundary condition
     (; hydrology_cm, θ_r, ν, S_s) = model.parameters
+    if typeof(hydrology_cm) <: ClimaCore.Fields.Field
+        hcm_bc = get_bottom_surface_field(hydrology_cm, surface_space)
+        θ_r_bc = get_bottom_surface_field(θ_r, surface_space)
+        ν_bc = get_bottom_surface_field(ν, surface_space)
+        S_s_bc = get_bottom_surface_field(S_s, surface_space)
+    else
+        hcm_bc = hydrology_cm
+        θ_r_bc = θ_r
+        ν_bc = ν
+        S_s_bc = S_s
+    end
+
     θ_bc = FT.(rre_bc.bc(p, t))
-    ψ_bc = @. pressure_head(hydrology_cm, θ_r, θ_bc, ν, S_s)
+    ψ_bc = @. pressure_head(hcm_bc, θ_r_bc, θ_bc, ν_bc, S_s_bc)
+    ψ_bc = Fields.Field(Fields.field_values(Fields.level(ψ_bc, 1)), axes(Δz))
```

Comment:
The purpose of this `Field` gymnastics is hard to understand. I don't know exactly what's the better way to this, but if you package this into a function a reader can guess your intent from the function name.

Reply:
I think this is no longer needed. This is what `get_bottom_surface_field` does. ill make sure it runs as expected.